<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EC2AMAZ-SFN1RDI" name="com.nbcd.Test.TCActivationPage_Mob_VerifyGenerallayout_3151" tests="2" failures="0" timestamp="10 Apr 2018 13:52:33 GMT" time="70.726" errors="1">
  <testcase name="Mob_VerifyGenerallayout_3151" time="70.726" classname="com.nbcd.Test.TCActivationPage_Mob_VerifyGenerallayout_3151"/>
  <testcase name="afterSuite" time="0.000" classname="com.nbcd.Test.TCActivationPage_Mob_VerifyGenerallayout_3151">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.nbcd.GenericLib.Extent_Reports.afterSuite(Extent_Reports.java:175)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- afterSuite -->
</testsuite> <!-- com.nbcd.Test.TCActivationPage_Mob_VerifyGenerallayout_3151 -->
